src/controllers/enter.information.controller.ts

```diff
@@ -0,0 +1,10 @@
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../model/template.paths";
+
+const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
```

Comment:
Out of interest does this have to be an async function with a promise?

Reply:
I think you only need the function to be async if you are await'ing something inside. in this case you can take off the async + Promise<void>.   You should be able to just use (blah, blah, blah) => {